bucket/megui.json

```
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://sourceforge.net/projects/megui/",
+    "description": "GUI based MPEG-4 encoding/muxing solution.",
+    "version": "2896",
+    "license": "GPL-2.0-or-later",
+    "url": "https://downloads.sourceforge.net/project/megui/megui-stable/2896/MeGUI-2896-32.zip",
+    "hash": "sha1:2c99eb83f0a27132b5942a09f81286886ee36b3e",
+    "bin": "MeGUI.exe",
+    "shortcuts": [
+        [
+            "MeGUI.exe",
+            "MeGUI"
+        ]
+    ],
+    "checkver": {
```

Try this as same green button is on homepage, with title containing file.

```suggestion
    "checkver": "MeGUI-(\\d+)-32\\.zip",
```